docs/source/en/model_doc/camembert.md

```diff
@@ -95,13 +98,61 @@ as the information relative to the inputs and outputs.
 
 ## CamembertForMultipleChoice
 
-[[autodoc]] CamembertForMultipleChoice
+You can also use CamemBERT for multiple-choice tasks via the command line:
```

Comment:
Add this to the <hfoption> blocks above and make sure the example matches that of the `Pipeline` and `AutoModel` for consistency. See the [BERT](https://huggingface.co/docs/transformers/model_doc/bert?usage=transformers+CLI) docs for an example